docs/supported-services/sns.rst

```diff
@@ -73,6 +73,35 @@ This Handel file shows an SQS service being configured:
           subscriptions:
             - endpoint: fake@example.com
               protocol: email
+              
+Example Handel File
+-------------------
+This Handel file shows an SNS Topic as a dependency to a Lambda Function
+
+.. code-block:: yaml
+
+    version: 1
+
+    name: departmentalAWSBilling
```

Comment:
Let's probably change the name to something more generic and remove our BYU-specific "tags" section.